ballista/rust/core/src/serde/logical_plan/mod.rs

```diff
@@ -1415,7 +1416,11 @@ mod roundtrip_tests {
         let schema = test_schema();
         let ctx = SessionContext::new();
         let options = CsvReadOptions::new().schema(&schema);
-        let df = ctx.read_csv(table_name, options).await?;
+
+        let uri = format!("file:///{}.csv", table_name);
+        ctx.register_csv(table_name, &uri, options).await?;
```

Comment:
Changing to using this method is necessary as otherwise it infers the `?table?` as the table name, which seems to result in an additional projection appearing, not entirely sure why though...

Reply:
CC @andygrove